control/strategy/pool.go

```diff
@@ -227,6 +230,9 @@ func (p *pool) applyPluginMeta(a AvailablePlugin) error {
 	case plugin.StickyRouting:
 		p.RoutingAndCaching = NewSticky(cacheTTL)
 		p.concurrencyCount = 1
+	case plugin.ConfigRouting:
+		p.RoutingAndCaching = NewConfigBased(cacheTTL)
+		p.concurrencyCount = 3
```

Comment:
Do you want to override the currency count here or allow it to be defined by the plugin author?